docs/quickstart_simple.md

```diff
@@ -67,37 +81,49 @@ cat > config.js
 Now open the config.js in your code editor and enter the following:
 
 ```Javascript
+require('dotenv').config()
+const HDWalletProvider = require("@truffle/hdwallet-provider");
+const fs = require("fs");
```

Comment:
consistency: use either single, or double quotes but never mix them all in one file. If you have prettier installed in your editor it should auto format code snippets in markdown files based on our eslint config